src/main/java/slimeknights/tconstruct/tables/tileentity/table/tinkerstation/ITinkerStationInventory.java

```diff
@@ -0,0 +1,14 @@
+package slimeknights.tconstruct.tables.tileentity.table.tinkerstation;
+
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.NonNullList;
+import slimeknights.mantle.recipe.inventory.IReadOnlyInventory;
+
+public interface ITinkerStationInventory extends IReadOnlyInventory {
+
+  NonNullList<ItemStack> getAllInputStacks();
+
+  void clearInputs();
```

Comment:
Should never be called by recipes. Keep in the wrapper, but remove from the interface